packages/modules/viewer-react/src/components/iModel/IModelLoader.tsx

```diff
@@ -63,10 +63,10 @@ const IModelLoader = React.memo((viewerProps: ModelLoaderProps) => {
 
     if (!requiredConnectionProps) {
       throw new Error(
-        IModelApp.localization.getLocalizedStringWithNamespace(
+        IModelApp.localization.getLocalizedString([
```

Comment:
whats with the change here??

Reply:
deprecated, was throwing errors, ill add it to description